core/src/main/scala/org/bykn/bosatsu/Package.scala

```diff
@@ -37,7 +37,8 @@ object Package {
   type PackageF2[A, B] = PackageF[A, A, B]
   type Parsed = Package[PackageName, Unit, Unit, Statement]
   type Resolved = FixPackage[Unit, Unit, (Statement, ImportMap[PackageName, Unit])]
-  type Inferred = FixPackage[NonEmptyList[Referant[Variance]], Referant[Variance], Program[TypeEnv[Variance], TypedExpr[Declaration], Statement]]
+  type Interface = FixPackage[NonEmptyList[Referant[Variance]], Referant[Variance], Unit]
```

Comment:
we can probably weaken this more: I think we only need the package name and the exports. I don't think we need the imports at all.